src/components/game/components/board/Board.js

```diff
@@ -24,90 +25,88 @@ class Board extends React.Component {
       const newDeck = this.state.deck.slice();
       const newPlayerHand = this.state.playerHand.slice();
       const newAiHand = this.state.aiHand.slice();
+      let newPlayerBet = this.state.playerBet;
+      let newAiBet = this.state.aiBet;
+      let newPhase = this.state.phase;
 
       newDeck.push(deckCreator());
       const newDeckFlat = newDeck.flat();
 
-      newPlayerHand.push(newDeckFlat[0]);
-      newPlayerHand.splice(0, 1);
-      newDeckFlat.splice(0, 1);
-      newPlayerHand.push(newDeckFlat[0]);
-      newDeckFlat.splice(0, 1);
-      newPlayerHand.push(newDeckFlat[0]);
-      newDeckFlat.splice(0, 1);
-
-      newAiHand.push(newDeckFlat[0]);
-      newAiHand.splice(0, 1);
-      newDeckFlat.splice(0, 1);
-      newAiHand.push(newDeckFlat[0]);
-      newDeckFlat.splice(0, 1);
-      newAiHand.push(newDeckFlat[0]);
-      newDeckFlat.splice(0, 1);
+      if (newPhase === 0) {
+        newPlayerHand.push(newDeckFlat[0]);
+        newPlayerHand.splice(0, 1);
+        newDeckFlat.splice(0, 1);
+        newPlayerHand.push(newDeckFlat[0]);
+        newDeckFlat.splice(0, 1);
+        newPlayerHand.push(newDeckFlat[0]);
+        newDeckFlat.splice(0, 1);
+        newPlayerHand.push(newDeckFlat[0]);
+        newDeckFlat.splice(0, 1);
+        newPlayerHand.push(newDeckFlat[0]);
+        newDeckFlat.splice(0, 1);
+
+        newAiHand.push(newDeckFlat[0]);
+        newAiHand.splice(0, 1);
+        newDeckFlat.splice(0, 1);
+        newAiHand.push(newDeckFlat[0]);
+        newDeckFlat.splice(0, 1);
+        newAiHand.push(newDeckFlat[0]);
+        newDeckFlat.splice(0, 1);
+        newAiHand.push(newDeckFlat[0]);
+        newDeckFlat.splice(0, 1);
+        newAiHand.push(newDeckFlat[0]);
+        newDeckFlat.splice(0, 1);
+
+        newPlayerBet += 100;
+        newAiBet += 100;
+        newPhase += 2;
```

Comment:
Avoid usage of += because it mutates variables. If needed create dedicated functions to do such operations